_2017_pages/faq.html

```diff
@@ -0,0 +1,56 @@
+---
+layout: 2017_default
+permalink: /faq
+title: Frequently Asked Questions
+description: "Answers to common questions attendees may have"
+---
+    <!--background color-->
+
+    <section id="conference" class="cfp cfp-container cfp-container-top">
+      <h1>{{ page.title }}</h1>
+      <!--
+      <div >
+         <a href ="https://forms.gle/uHcTE53bpUYPHHJ79" class="btn btn-sm normal-text">Click here to submit</a>
+      </div>
+      --!>
+
+      <h2 class="first">Key Dates</h2>
+      <p>Async Conference Start: October 24th on PyMC Discourse</p>
+      <p>Beginner Zone: October 24th to October 7th on Zulip </p>
+      <p>Synchronous Conference: October 31st on Webex and Zulip</p>
+
+      <h2 class="first">Do I need to purchase a ticket?</h2>
+      <p> For the async beginner zone you <b>do not</b> need to purchase a ticket. These are
+        for the good of the community and open to anyone that would benefit.
+
+        A ticket is required for the synchronous conference.
```

Comment:
I would change this to something like:

```

<p>A ticket is only required for the synchronous conference.</p>

<p> To take part in any of the async activities such as the beginner zone or the talk discussion on Discourse you <b>do not</b> need to purchase a ticket.</p>

```

I removed the for the good of the community part too as I think both sync and async are for the good of the community